src/commons/Reporter.ts

```diff
@@ -240,6 +241,13 @@ export namespace Reporter {
     allureReporter.addDescription(description, descriptionType);
   }
 
+  /**
+   * Adding screenshot for report
+   * @param name of the screenshot, that will appear in the report
+   */
+  export function addAttachment(name: string = 'screenshot'): void {
```

Comment:
the function name should reflect what it does. how about addScreenshot instead?

Reply:
@FelixZilber yes, I agree with you